packages/plugin-compat/sources/extensions.ts

```diff
@@ -149,4 +149,17 @@ export const packageExtensions: Array<[string, any]> = [
       "react-dom": `^15.0.0 || ^16.0.0`,
     },
   }],
+  // https://github.com/react-bootstrap-table/react-bootstrap-table2/pull/1491
+  [`react-bootstrap-table2-paginator@*`, {
+    dependencies: {
+      "classnames": "^2.2.6"
+    },
+  }],
+  // https://github.com/STRML/react-draggable/pull/525
+  [`react-draggable@*`, {
+    dependencies: {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+    },
```

Comment:
These should most likely be peer dependencies

Reply:
Hi @merceyz,

The error I got in PnP while I was working with that package mentioned that the package was looking for react and react-dom as dependencies. But if you think that adding them in peerDependencies makes sense, then I will make that change 👍

Reply:
Yeah, `react` and `react-dom` are usually peer dependencies but PnP doesn't know that, it just knows it's undeclared

Reply:
Understood. Changes made upstream :)